PcAtChipsetPkg/Test/Library/MockResetSystemLib/MockResetSystemLib.c

```diff
@@ -0,0 +1,40 @@
+/** @file

```

Comment:
I think this version isn't needed? We should just have the one version in MdeModulePkg, right?